tests/autorestart_test/test_container_autorestart.py

```diff
@@ -18,16 +18,17 @@ def get_autorestart_container_list(duthost):
     """
     @summary: Get container list by analyzing the command output of 
               "show container feature autorestart"
-    @return: The list includes containers which were implemented with the autorestart feature
+    @return: The list includes containers which was implemented with the autorestart feature
```

Comment:
Rephrase: `@return: A list of all containers which have the autorestart feature implemented`

Reply:
Will update.